util/ubuntuinstall.bash

```diff
@@ -1,32 +1,37 @@
 #!/bin/bash
 
-echo "This script installs deps for running depserver on a Ubuntu server."
```

Comment:
perhaps you could create a `ubuntu2204-install.bash` with your new changes, so we support older installations too.

Reply:
fixed in earlier commits 052a11c a5beab0